lib/recurly/resources/CouponDiscount.js

```diff
@@ -14,15 +14,15 @@ const Resource = require('../Resource')
  * @typedef {Object} CouponDiscount
  * @prop {Array.<CouponDiscountPricing>} currencies - This is only present when `type=fixed`.
  * @prop {number} percent - This is only present when `type=percent`.
- * @prop {Object} trial - This is only present when `type=free_trial`.
```

Comment:
Was this taken out because it's no longer true or because we don't want it in the in-line documentation?

Reply:
possibly missing the `description` property

Reply:
I'll check in a follow up